generators/ci-cd/template/.gitlab-ci.yml

```diff
@@ -0,0 +1,37 @@
+image: mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
your ci is invalid you can fork and try it on this (https://gitlab.com/nicolas63/jhipster-sample-app-dotnetcore)